src/controllers/SalesEmployeeController.ts

```diff
@@ -0,0 +1,16 @@
+import express from "express";
```

Comment:
Can be `import  {Request, Response } from "express"`

So like 4 and 8 can be  `async (req: Request, res: Response):`